taxi/forms.py

```diff
@@ -0,0 +1,30 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
```

Comment:
The `DriverCreationForm` should include validation for the `license_number` field to ensure it consists of 3 uppercase letters followed by 5 digits, as specified in the task description. Consider adding a custom clean method or a validator to enforce this rule.